pkgs/tools/nix/nixdoc/default.nix

```diff
@@ -2,38 +2,24 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "nixdoc";
-  version = "1.0.1";
+  version = "2.2.0-git+61f63fab";
```

Comment:
```suggestion
  version = "unstable-2023-07-01";
```

Reply:
It would be `2.2.0-unstable-2023-07-01` according to the [current policy](https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming).